### PR TITLE
Support to save default value to expression

### PR DESCRIPTION
**Pull request by laura-ding**

### CODE REVIEW

**Review by dutor — DISMISSED** [review]
lGTM.

**Review by jievince — APPROVED** [review]
LGTM

File: src/common/clients/meta/MetaClient.cpp [hunk 1]
```diff
@@ -287,19 +287,33 @@ bool MetaClient::loadSchemas(GraphSpaceID spaceId,
...
-            bool hasDef = colIt.__isset.default_value;
-            auto& colType = colIt.get_type();
-            size_t len = colType.__isset.type_length ? *colType.get_type_length() : 0;
-            bool nullable = colIt.__isset.nullable ? *colIt.get_nullable() : false;
-            schema->addField(colIt.get_name(),
-                             colType.get_type(),
-                             len,
-                             nullable,
-                             hasDef ? *colIt.get_default_value() : Value());
+        for (auto colIt : tagIt.schema.get_columns()) {
```
Comment (yixinglu) [inline src/common/clients/meta/MetaClient.cpp:315]: `auto& colIt`

File: src/common/clients/meta/MetaClient.cpp [hunk 2]
```diff
@@ -331,16 +345,8 @@ bool MetaClient::loadSchemas(GraphSpaceID spaceId,
...
-            bool hasDef = col.__isset.default_value;
-            auto& colType = col.get_type();
-            size_t len = colType.__isset.type_length ? *colType.get_type_length() : 0;
-            bool nullable = col.__isset.nullable ? *col.get_nullable() : false;
-            schema->addField(col.get_name(),
-                             colType.get_type(),
-                             len,
-                             nullable,
-                             hasDef ? *col.get_default_value() : Value());
+        for (auto col : edgeIt.schema.get_columns()) {
```
Comment (yixinglu) [inline src/common/clients/meta/MetaClient.cpp:348]: ditto

**Review by yixinglu — APPROVED** [review]
LGTM

**Review by jievince — APPROVED** [review]

**Pull request merged**